scripts/run-operator.sh

```diff
@@ -15,9 +15,13 @@ then
   minikube start
 fi
 
-echo "building operator image in minikube"
+GIT_HASH="$(git rev-parse HEAD)"
+TMP_INSTALL_DIR="$(mktemp -d)"
+trap 'rm -rf -- "$TMP_INSTALL_DIR"' EXIT
+
+echo "building operator image in minikube for commit ${GIT_HASH}"
 KROXYLICIOUS_VERSION=${KROXYLICIOUS_VERSION:-$(mvn org.apache.maven.plugins:maven-help-plugin:3.4.0:evaluate -Dexpression=project.version -q -DforceStdout)}
-minikube image build . -f Dockerfile.operator -t quay.io/kroxylicious/operator:latest --build-opt=build-arg=KROXYLICIOUS_VERSION="${KROXYLICIOUS_VERSION}"
+minikube image build . -f Dockerfile.operator -t quay.io/kroxylicious/operator:${GIT_HASH} --build-opt=build-arg=KROXYLICIOUS_VERSION="${KROXYLICIOUS_VERSION}"
```

Comment:
There may be some minor value in using a tag which makes it obvious that this is a local build, or at least that this checksum is something meaningful to `git`.